cli/bench/types.go

```diff
@@ -8,3 +13,44 @@ type Results struct {
 	duration             float64 // Benchmark duration.
 	requestsPerSecond    int     // Cumber of requests per second - the main measured value.
 }
+
+// Interface for various types of requests
+type Request interface {
+	execute()
+}
+
+// InsertRequest data structure for insert request
+type InsertRequest struct {
+	ctx                 context.BenchCtx
+	tarantoolConnection *tarantool.Connection
+	results             *Results
+}
+
+// SelectRequest data structure for select request
+type SelectRequest struct {
+	ctx                   context.BenchCtx
+	tarantoolConnection   *tarantool.Connection
+	results               *Results
+	getRandomTupleCommand string
+}
+
+// UpdateRequest data structure for update request
+type UpdateRequest struct {
+	ctx                   context.BenchCtx
+	tarantoolConnection   *tarantool.Connection
+	results               *Results
+	getRandomTupleCommand string
+}
+
+// RequestsPool data structure for abstraction of a heap of identical requests
+type RequestsPool struct {
+	request Request // InsertRequest, SelectRequest or UpdateRequest
+	count   int     // Count of requests
+}
+
+// RequestsSequence data structure for abstraction for the constant issuance of new requests
+type RequestsSequence struct {
+	requests            []RequestsPool
+	currentRequestIndex int // currentRequestIndex describes what type of request will be issued by the sequence
```

Comment:
Set a dot at the end of the sentence.

Reply:
The string length is greater than 100. Why this is bad described in the previous review.